metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/utils/EntityMergeEngine.java

```diff
@@ -345,35 +346,34 @@ private static void convertAndAddEntity(RDF rdf, EnrichmentBase enrichmentBase,
       throw new IllegalArgumentException("Unknown entity type: " + enrichmentBase.getClass());
     }
 
-    // Append it to the proxy if needed.
-    if (entity != null && StringUtils.isNotEmpty(fieldName)) {
-      RdfProxyUtils.appendToProxy(rdf, entity, fieldName);
+    // Append it to the europeana proxy if needed.
+    if (entity != null && StringUtils.isNotBlank(fieldName)) {
+      RdfProxyUtils.appendToEuropeanaProxy(rdf, entity, fieldName);
     }
   }
 
   /**
-   * Merge entities in a record. Wrapper for {@link #mergeEntities(RDF, List, String)} without a
-   * field name.
+   * Merge entities in a record. Wrapper for {@link #mergeEntity(RDF, EnrichmentBase, String)}
+   * without a field name.
```

Comment:
I think this description is no longer quite valid?

Reply:
Yes and no, vague description. Removed the last part anyhow.